multi-node.sh

```diff
@@ -8,6 +8,7 @@ slave_value=${!slave_name}
 ami=($slave_value)
 NODES=2
 tmp_script=$(mktemp -p $WORKSPACE)
```

Comment:
Just to confirm the behavior. This line is overriding the value created in common.sh
```
tmp_script=${tmp_script:-$(mktemp -p $WORKSPACE)}
```
That will cause two tmp files to be made, was this the behavior you wanted?

Reply:
No, it will set tmp_script if unset. See this doc: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html